docs/reference/commandline/service_create.md

```diff
@@ -245,8 +245,8 @@ metadata](https://docs.docker.com/engine/userguide/labels-custom-metadata/).
 
 ### Add bind-mounts or volumes
 
-Docker supports two different kinds of mounts, which allow containers to read to
-or write from files or directories on other containers or the host operating
+Docker supports two different kinds of mounts, which allow containers to read
```

Comment:
Ah, I think `tmpfs` wasn't functional at the time of writing, so likely wasn't updated when it became functional